### ISSUE TITLE
Annotations is removed is latest run

### ISSUE THREAD

**Issue by arahuja** [hammerlab/cycledash#740]
![image](https://cloud.githubusercontent.com/assets/455755/8172826/0a83f77e-1393-11e5-81e1-2056561cee6f.png)

**Comment by ihodes** [comment]
Appears to be there on (at least some) old runs (e.g. http://cycledash.demeter.hpc.mssm.edu/runs/270/examine?query=ORDER+BY+contig%2C+position), but isn't running anymore? Is the worker queue up still @danvk?

**Comment by arahuja** [comment]
The gene_names were there before I refreshed to the latest deploy, this was submitted before the update.

**Comment by arahuja** [comment]
So I did have them in the new deploy in another tab, refreshed that tab and now they are gone.

**Comment by arahuja** [comment]
And now back: 
![image](https://cloud.githubusercontent.com/assets/455755/8172906/e75eb9e0-1393-11e5-8ae1-264a0e483776.png)

**Comment by ihodes** [comment]
Mind linking the run that's exhibiting this behavior?

**Comment by arahuja** [comment]
...and now gone: http://cycledash.demeter.hpc.mssm.edu/runs/327/examine?query=sample_name+%3D+TUMOR+ORDER+BY+contig%2C+position

**Comment by arahuja** [comment]
Verifying I'm not crazy:
![image](https://cloud.githubusercontent.com/assets/455755/8172955/9cfca56e-1394-11e5-847c-3486e24a82f1.png)

<refresh>

![image](https://cloud.githubusercontent.com/assets/455755/8172957/a41134f0-1394-11e5-8b71-47048a09779c.png)

**Comment by ihodes** [comment]
I just reproduced it. By refreshing several times.

**Comment by danvk** [comment]
Did you guys figure out what was going on last night? I'm seeing an annotations column on every examine page linked to in this issue.

I do see cycledash-worker tasks running:

```
$ sudo initctl list
cycledash-worker (3) start/running, process 18407
cycledash-worker (2) start/running, process 18402
cycledash-worker (1) start/running, process 18398
```

**Comment by ihodes** [comment]
Try refreshing the page a bunch; its appearance is totally random.

**Comment by danvk** [comment]
I shift-reloaded Arun's page a few times and saw it. Fun!

**Comment by ihodes** [comment]
Yeah, quite :) I'm more than happy to look at this when I have a few hours. Haven't seen it locally, so it may need to wait until I'm back unless one of y'all get to it.

**Comment by danvk** [comment]
I'm not seeing anything in the flask logs.

**Comment by ihodes** [comment]
Yeah my guess is this could be related to the race conditions in RecordStore.

**Comment by ihodes** [comment]
I'm not sure how, though. I'd bet the backend is functioning just fine though.

**Comment by danvk** [comment]
Looking at the network traces supports that theory: even when the annotations column isn't visible, I see it mentioned both in the examine HTML and in the `/genotypes` response.

**Comment by ihodes** [comment]
The store needs a total rewrite to be safe anyway. This may be the forcing function.
